docs/fundamentals/read-operations.txt

```diff
@@ -269,6 +271,106 @@ documents.
+results. By default, the ``orderBy()`` method sets an ascending sort on
+the supplied field name, but you can explicitly specify an ascending
+sort by passing ``'asc'`` as the second parameter. To
+specify a descending sort, pass ``'desc'`` as the second parameter.
+
+If your documents contain duplicate values in a specific field, you can
+handle the tie by specifying additional fields to sort on. This ensures consistent
+results if the additional fields contain unique values.
+
+This example queries for documents in which the value of the ``countries`` field contains
+``'Indonesia'`` and orders results first by an ascending sort on the
+``year`` field, then a descending sort on the ``title`` field.
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      Use the following syntax to specify the query:
+
+      .. code-block:: php
+
+         $movies = Movie::where('countries', 'Indonesia')
+             ->orderBy('year')
+             ->orderBy('title', 'desc')
+             ->get();
+
+   .. tab:: Controller Method
+      :tabid: controller
+
+      To see the query results in the ``browse_movies`` view, edit the ``show()`` function
+      in the ``MovieController.php`` file to resemble the following code:
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: php
+
+            class MovieController
+            {
+                public function show()
+                {
+                    $movies = Movie::where('countries', 'Indonesia')
+                        ->orderBy('year')
+                        ->orderBy('title', 'desc')
+                        ->get();
+
+                    return view('browse_movies', [
+                        'movies' => $movies
+                    ]);
+                }
+            }
+
+         .. output::
+            :language: none
+            :visible: false
+
+            Title: Joni's Promise
+            Year: 2005
+            Runtime: 83
+            IMDB Rating: 7.6
+            IMDB Votes: 702
+            Plot: A film delivery man promises ...
+            
+            Title: Gie
+            Year: 2005
+            Runtime: 147
+            IMDB Rating: 7.5
+            IMDB Votes: 470
+            Plot: Soe Hok Gie is an activist who lived in the sixties ...
+            
+            Title: Requiem from Java
+            Year: 2006
+            Runtime: 120
+            IMDB Rating: 6.6
+            IMDB Votes: 316
+            Plot: Setyo (Martinus Miroto) and Siti (Artika Sari Dewi)
+            are young married couple ...
+            
+            ...
+
+.. tip::
+
+   To learn more about sorting, see the following resources:
+
+   - :manual:`Natural order </reference/glossary/#std-term-natural-order>`
+     in the Server manual glossary
+   - `Ordering, Grouping, Limit and Offset <https://laravel.com/docs/10.x/queries#ordering-grouping-limit-and-offset>`__
```

Comment:
The Laravel version in the doc url needs to be updated to 11.x for laravel-mongodb 4.2